src/other/Table.js

```diff
@@ -307,27 +325,40 @@
             .append("td")
         ;
         cells[this.renderHtmlDataCells() ? "html" : "text"](function (d, idx) { 
-            var retVal = context.columnPatterns()[idx] ? context.getColumnFormatting(d, idx) : d;
+            var retVal;
+            if (!(d instanceof Widget)) {
+                retVal = context.columnPatterns()[idx] ? context.getColumnFormatting(d, idx) : d;
+            }
             return retVal; 
```

Comment:
undefined and null are probably handled the same way - so returning "" would probably be the wrong thing to do?